lib/types/string/index.js

```diff
@@ -188,7 +191,9 @@ internals.String = class extends Any {
             options = this._addressOptions(options);
         }
 
-        return this._rule('email', { args: { options } });
+        const regex = /\s*,\s*/i;
```

Comment:
Can you add an option to customize the separator?

Reply:
ok, I will do it.